Screens/Programs.js

```diff
@@ -76,12 +80,20 @@ function Programs({ navigation }) {
 
           <SafeAreaView>
             <ScrollView style={styles.scrollView}>
-              {programList.map((item) => (
+              {programList.map((item, index) => (
                 <View key={item.name}>
                   <Pressable
+                    style={styles.itemButton}
                     onPress={() =>
                       navigation.navigate("COURSE", {
                         ID: item.name,
+                        Desc: item.description,
+                        InPer: item.inperson,
+                        Online: item.online,
+                        StartTime: item.start,
+                        EndTime: item.end,
+                        Weekday: item.weekday,
+                        Index: index,
```

Comment:
if programList is no longer a global array and all info is being passed anyways, why is index being passed to next page too?

Reply:
These keys are passed to course.js for displaying contents for each program. And index is kept temporarily since I am still finding a way to represent weekday information for each program.